tests/provision/beaker/bootc.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+    rlPhaseStartSetup
+        rlRun "pushd data"
+        rlRun "set -o pipefail"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Check bootc works well"
+        rlRun -s "tmt run --dry plan --name plan/bootc$"
+
+        rlAssertGrep 'ostreecontainer --url quay.io/fedora/custom-bootc:latest' $rlRun_LOG
+        rlAssertGrep 'dummysecret' $rlRun_LOG
+        rlAssertGrep '{"auths": {"quay.io": {"auth": "dummysecret"}}}' $rlRun_LOG
+    rlPhaseEnd
+
```

Comment:
```suggestion
    echo '' > $rlRun_LOG

```
I think `rlRun_LOG` should be cleared after this test because if the test below fails, the grep statements will still pass. The log will still contain the output from this test.

Reply:
> rlRun_LOG is cleared with every rlRun -s, all tmt calls are through rlRun -s, so we should be safe here.

didn't see the above comment until just now:)